.circleci/config.yml

```diff
@@ -25,7 +29,7 @@ workflows:
             branches:
               ignore: /.*/
 jobs:
-  test-3.6: &test-template
+  test-36: &test-template
```

Comment:
any reason for this change?

Reply:
I'm not familiar with circleci, but from what I could tell, version 2.1 here (which I needed to update to for the Windows orb) doesn't allow `.`s in the names of jobs

Reply:
That's OK then.